0016-pre-shared-key/0016-pre-shared-key.md

```diff
@@ -182,12 +145,11 @@ encrypted_data, auth_tag = aes_256_gcm({
   data: private_headers
 })
 
-// Now the PSK data object is constructed, containing the public headers, the
-// encrypted private headers, and the encrypted application data. The
-// "public_headers" are added, including a "key" header that specifies the HMAC
-// algorithm used (which MUST be hmac-sha-256) and the nonce used to generate
-// the payment's encryption key. The AES-256-GCM authentication tag is attached
-// to the 'encryption' header after a space.
+// The PSK data object is constructed from the public headers, encrypted private
+// headers, and the encrypted application data. The "public_headers" include a
+// "key" header that specifies the HMAC algorithm used (which MUST be hmac-sha-256)
```

Comment:
In PSK 1.0 the default should actually be no Key header, and instead to send a rejection message for incoming payments with Key headers.